pages/favoritespots.js

```diff
@@ -27,7 +27,7 @@ export default function favoritespots({ favoriteSpots, surfspots }) {
   return (
     <>
       <Head>
-        <title>SpotiFinder</title>
+        <title>SpotGuide</title>
       </Head>
```

Comment:
Ich glaube, du brauchst den Head nicht auf jeder einzelnen Seite. Soweit ich weiß, reicht das, wenn du den in der _app.js hast, dann wird auf jeder Unterseite der Titel im Browser angezeigt